template/.github/workflows/integration-test.yml

```
@@ -15,7 +15,8 @@ on:
           - profile
           - custom
       test-mode-input:
-        description: The profile or the runner used
+        description: |
+          The profile or the runner used. Eg: `amd64` or `smoke-latest` (see test/interu.yaml)
```

suggestion: I would switch the order of the examples to be the same as in the first sentence.

```suggestion
          The profile or the runner used. Eg: `smoke-latest` or `amd64` (see test/interu.yaml)
```